pkg/minikube/bootstrapper/bsutil/kubeadm.go

```diff
@@ -105,6 +105,10 @@ func GenerateKubeadmYAML(k8s config.KubernetesConfig, r cruntime.Manager) ([]byt
 	if version.GTE(semver.MustParse("1.14.0-alpha.0")) {
 		configTmpl = template.KubeAdmConfigTmplV1Beta1
 	}
+	// v1beta2 isn't required until v1.18.
+	if version.GTE(semver.MustParse("1.18.0-alpha.0")) {
```

Comment:
The comment for this PR and the comments in the linked issue say that v1beta2 should be used for v1.17 and higher, so this should parse `1.17.0` instead of `1.18.0-alpha.0`

Reply:
@priyawadhwa I made the change, thank you.